spec/node/asciidoctor.spec.js

```diff
@@ -9,11 +9,20 @@ var config = {
   }
 };
 var asciidoctor = require('../../build/asciidoctor.js')(config);
+function asciidoctorVersionGreaterThan (version) {
```

Comment:
Btw, I tried to adopt the semantic version pattern in asciidoctor core, but rubygems.org rejects it. So it's actually impossible as far as I can tell to use the semantic version pattern (specifically the hyphen) for a gem...unless there's something I missed.